pkg/cri/sbserver/sandbox_stats_windows.go

```diff
@@ -416,6 +426,9 @@ func (c *criService) getSandboxPidCount(ctx context.Context, sandbox sandboxstor
 
 		processes, err := task.Pids(ctx)
 		if err != nil {
+			if errdefs.IsNotFound(err) {
```

Comment:
this requires https://github.com/microsoft/hcsshim/pull/1807 but should be ok to merge this and follow up via hcsshim version bump.  Alternatively, I can handle the errors directly here.